locations/spiders/lubaszka_pl.py

```diff
@@ -0,0 +1,37 @@
+from scrapy import Spider
+
+from locations.hours import DAYS_PL, OpeningHours
+from locations.items import Feature
+
+
+class LubaszkaPLSpider(Spider):
+    name = "lubaszka_pl"
+    item_attributes = {"brand": "Galeria Wypieków Lubaszka", "brand_wikidata": "Q108586693"}
+    start_urls = ["https://www.lubaszka.pl/sklepy"]
+    custom_settings = {"ROBOTSTXT_OBEY": False}  # robots.txt causes 404 redirect loop
+
+    def parse(self, response, **kwargs):
+        for shopId in response.xpath("//div[contains(@id, 'address_')]/@id").getall():
+            streetAddress = response.xpath(f"//div[@id='{shopId}']/div[@class='row']/div[1]/text()").getall()[0].strip()
```

Comment:
Minor suggestion for future spiders: `getall()[0]` is lengthier than `get()` (only first result returned)